project_timesheet_time_control/models/account_analytic_line.py

```diff
@@ -38,7 +38,7 @@ def _running_domain(self):
         """Domain to find running timesheet lines."""
         return [
             ("date_time", "!=", False),
-            ("employee_id", "in", self.env.user.employee_ids.ids),
+            ("user_id", "=", self.env.user.id),
```

Comment:
I remembered an error message when trying to add a timesheet with a user with no employee, but it seems that message has gone away.

Anyway, the duality is still there, so please put both conditions:

```suggestion
            '|',
            ("employee_id", "in", self.env.user.employee_ids.ids),
            ("user_id", "=", self.env.user.id),
```

Reply:
In the timesheet lines, `user_id` and `employee_id` are always consistent.
See https://github.com/odoo/odoo/blob/13.0/addons/hr_timesheet/models/hr_timesheet.py#L134-L137 defined in `_timesheet_preprocess()` and invoked in create and write.
Also see https://github.com/odoo/odoo/blob/13.0/addons/hr_timesheet/models/hr_timesheet.py#L72-L78 in the create.

Having only `("user_id", "=", self.env.user.id)` is just fine IMO.

Reply:
No if you don't have any user linked to your employee. Both searches are needed.

Reply:
Thinking twice, although that can happen, not for "interactive" timesheeting, so OK to the switch.